### ISSUE TITLE
hdd #27 preview missing

### ISSUE THREAD

**Issue by JustLuuuu** [kodadot/nft-gallery#10200]
https://kodadot.xyz/ahp/gallery/127-2405100695

### Only preview missing
<img width="974" alt="Screenshot 2024-05-03 at 11 13 00" src="https://github.com/kodadot/nft-gallery/assets/104380776/519b8369-da02-4593-92cb-3eef514e4724">

### The image is there:
<img width="839" alt="Screenshot 2024-05-03 at 11 14 15" src="https://github.com/kodadot/nft-gallery/assets/104380776/4f613282-a705-4153-a94c-5373edf02c32">

**Comment by JustLuuuu** [comment]
Same with this NFT: https://kodadot.xyz/ahp/collection/121
<img width="1234" alt="Screenshot 2024-05-03 at 11 42 40" src="https://github.com/kodadot/nft-gallery/assets/104380776/6b1a775c-4c8f-4fa4-b7d5-b8a70de3a98c">

**Comment by preschian** [comment]
Related: https://github.com/kodadot/nft-gallery/issues/10188. Let's continue there. 

tldr; both of them should not pass `/code-checker`. Electronik drops was webgl. ref: https://github.com/kodadot/nft-gallery/issues/10144#issuecomment-2088078040

Need add more checker: https://github.com/kodadot/nft-gallery/issues/10207. For a temporary fix, I can manually upload the image.
